postgres/server/init.sls

```diff
@@ -214,6 +211,18 @@ postgresql-pg_ident:
       - service: postgresql-running
       {%- endif %}
 
+backup_pg_ident:
```

Comment:
Like above, we need the `if` statement here. These states also should be applied only when `postgres.config_backup` variable would contain any meaningful value.